CPAC/resources/configs/pipeline_config_rbc-options.yml

```
@@ -79,7 +79,7 @@ functional_preproc:
 
   motion_estimates_and_correction:
     motion_correction:
-      using: [3dvolreg]
+      using: [mcflirt]
```

```suggestion
      using: [3dvolreg]
```

Reverting since the problem exists with only 4 participants.